.github/workflows/docs_stable.yml

```
@@ -0,0 +1,38 @@
+name: Stable Docs Publish
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  deploy:
+    if: github.repository_owner == 'Qiskit-Extensions'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -U virtualenv setuptools wheel tox
+          sudo apt-get install graphviz pandoc
+      - name: Build docs stable
+        env:
+          QISKIT_DOCS_BUILD_TUTORIALS: 'always'
+        run: EXPERIMENTS_DEV_DOCS=1 PROD_BUILD=1 tox -e docs
```

Nit: we can remove this environment variables
```suggestion
        run: tox -e docs
```